logbook-json/src/main/java/org/zalando/logbook/json/PrettyPrintingJsonBodyFilter.java

```diff
@@ -16,16 +19,14 @@
 @Slf4j
 public final class PrettyPrintingJsonBodyFilter implements BodyFilter {
 
-    private final ObjectMapper mapper;
-    private final ObjectWriter writer;
+    private final JsonFactory factory;
 
-    public PrettyPrintingJsonBodyFilter() {
-        this(new ObjectMapper());
+    public PrettyPrintingJsonBodyFilter(final JsonFactory factory) {
+        this.factory = factory;
     }
 
-    public PrettyPrintingJsonBodyFilter(final ObjectMapper mapper) {
```

Comment:
Keeping signature compatibility should be an option here, right?

Reply:
Sure, hold on for update.